modules/applications/pages/download-and-launch.adoc

```diff
@@ -70,6 +70,11 @@ Two environments are available:
 The first time, the initialisation can take a while, a waiting message appears. Reload the page if the page is blank.
 ====
 
+[TIP]
+====
+If encountering issues Windows and WSL, check our xref:applications:faq.adoc#_i_am_runing_bonita_with_windows_and_bonita_ui_builder_with_wsl_does_not_work[FAQ] .
```

Comment:
:x: With this syntax, if we rename the title in other page, they will break this link.
Prefer use an internal anchor with this synthax [#WSLissue] to avoid the previous issue